vespalib/src/vespa/vespalib/io/fileutil.cpp

```diff
@@ -433,7 +433,7 @@ bool
 File::unlink()
 {
     close();
-    return vespalib::unlink(_filename);
+    return std::filesystem::remove(std::filesystem::path(_filename));
```

Comment:
Since the legacy code throws our internal `IoException` exception type, we might want to do a sweep of our code to see if we have any explicit `catch (vespalib::IoException)` or `catch (vespalib::Exception)` blocks for graceful IO error handling anywhere that should be upgraded to use the stdlib exception types.